src/main/java/nl/knaw/dans/bridge/plugin/lib/common/DarPluginConf.java

```diff
@@ -67,30 +81,9 @@ public void setActionClassName(String actionClassName) {
     public void setXslSourceList(List<XslSource> xslSourceList) {
         this.xslSourceList = xslSourceList;
     }
-/*
-    enum Type {
-        ARCHIVER("ARCHIVE"),
-        REGISTRAR("REGISTRAR");
-        IRODS("IRODS");
-        private String value;
-
-        Type(String value) {
-            this.value = value;
-        }
-
-        public String toString() {
-            return String.valueOf(value);
-        }
-
-        public static Type fromValue(String text) {
-            for (Type b : Type.values()) {
-                if (String.valueOf(b.value).equals(text)) {
-                    return b;
-                }
-            }
-            throw new IllegalArgumentException("No matching constant for [" + text + "]");
-        }
-    }
-    */
+    /*
```

Comment:
Maybe we should make a note for this 'type' of dar's that we don't model now, and then remove all this comment

Reply:
Ok, I delete it